glide.yaml

```diff
@@ -34,22 +36,22 @@ import:
 - package: k8s.io/apiextensions-apiserver
   version: 3eb4c5fefbe7ad866747f78c7e4994cb555170c6
 - package: github.com/prometheus/client_golang
-  version: v0.8.0
+  version: e7e903064f5e9eb5da98208bae10b475d4db0f8c
   subpackages:
   - prometheus
   - prometheus/promhttp
 - package: github.com/jbw976/go-ps
   version: 82859aed1b5dafbbe6ad4f3dac5404d2a742cf18
-testImport:
-- package: github.com/ghodss/yaml
-  version: 73d445a93680fa1a78ae23a5839bad48f32ba1ee
 - package: github.com/aws/aws-sdk-go
```

Comment:
what part of your changes need aws-sdk-go to no longer be a testImport? we are using it in product code now?